### PR TITLE
Add inorganic compounds batch 3

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/interfaces/utils.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export interface Qualitative<T> {
 }
 
 export interface Measure {
-  value: number;
+  value: number | number[];
```
Comment (maneetgoyal) [inline src/interfaces/utils.ts:15]: Revert the change. 

Instead try this:

```ts
export interface PhysicalProperties {
  name: string;
  formula: string;
  formulaWeight: number;
  color?: Qualitative<Colors>[] | Qualitative<Colors>;
  crystallineForms?:
    | Qualitative<CrystallineForms>[]
    | Qualitative<CrystallineForms>;
  refractiveIndex?: Measure | MeasureRange | Measure[];  // CHANGE
  specificGravity?: SpecificGravity;
  meltingOutcomes: MeltingOutcome[];
  boilingOutcomes: MeltingOutcome[];
  solubilitySheet: SolubilitySheet;
}
```

File: src/data/inorganic-compounds.ts [hunk 2]
```diff
@@ -916,4 +916,430 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    formula: 'NH3',
+    formulaWeight: 17.03,
+    color: {
+      nature: Colors.Colorless,
+    },
+    crystallineForms: {
+      nature: CrystallineForms.Gas,
+    },
+    refractiveIndex: { value: 1.325, units: 'lq' },
+    specificGravity: {
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts:931]: Ammonia has 2 specific gravities

File: src/data/inorganic-compounds.ts [hunk 3]
```diff
@@ -916,4 +916,430 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    name: 'Ammonium acetate carbonate, sesqui-',
+    formula: '(NH4)2CO3.2NH4HCO3.H2O',
+    formulaWeight: 272.21,
+    color: { nature: Colors.White },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        value: 25,
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts:1211]:
```suggestion
        value: 20,
```

File: src/data/inorganic-compounds.ts [hunk 4]
```diff
@@ -916,4 +916,430 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+        nature: SolubilityNature.Insoluble,
+        value: 1.25,
+        temperature: {
+          value: 100,
+          units: 'C',
+        },
+        gas: false,
+      },
+      ethylAlcohol95percent: {
+        nature: SolubilityNature.Soluble, // extra info is 0.005 al (ethylAlcohol95percent)
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts:1321]: Quantitative solubility should be used instead

File: src/data/inorganic-compounds.ts [hunk 5]
```diff
@@ -916,4 +916,430 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+      },
+    },
+  },
+
+  AmmoniumTetraChloroplatinate: {
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts:1326]:
```suggestion
  AmmoniumChloroplatinite: {
```

File: src/data/inorganic-compounds.ts [hunk 6]
```diff
@@ -916,4 +916,430 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
...
+    name: 'Ammonium chloroplatinate',
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts:1327]:
```suggestion
    name: 'Ammonium chloroplatinite',
```

**Review by maneetgoyal — COMMENTED** [review]
Few corrections, rest LGTM 👍

**Comment by sumitnegi7** [comment]
Sir @maneetgoyal  have done the suggested corrections please have a look.

File: src/data/inorganic-compounds.ts [hunk 7]
```diff
@@ -1318,13 +1324,14 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
         gas: false,
       },
       ethylAlcohol95percent: {
-        nature: SolubilityNature.Soluble, // extra info is 0.005 al (ethylAlcohol95percent)
+        nature: SolubilityNature.Soluble,
+        formsEntity: '0.005 ethylAlcohol95percent',
```
Comment (maneetgoyal) [inline src/data/inorganic-compounds.ts]: Use quanititative solubitiy!!

**Review by maneetgoyal — CHANGES_REQUESTED** [review]

File: src/data/inorganic-compounds.ts (hunk 7 above)
Comment (maneetgoyal) [reply src/data/inorganic-compounds.ts]:
```ts
export interface QuantitativeSolubility {
  value: number | string;
  temperature?: Measure;
  gas: boolean;
}
```

Comment (sumitnegi7) [reply src/data/inorganic-compounds.ts]: Sorry sir

**Review by maneetgoyal — APPROVED** [review]
LGTM

**Pull request merged**